docs/pages/upgrading/automatic-agent-updates.mdx

```diff
@@ -0,0 +1,242 @@
+---
+title: Set up Automatic Agent Updates
+description: Describes how to set up automatic agent updates.
```

Comment:
Since this is specifically for Cloud users, should we mention this in the title/description?